.changeset/famous-hats-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+`Tooltip`s no longer displayed upon hovering out of the reference element after clicking it. Components that use `Tooltip` internally are also affected. (E.g.`IconButton`, `SideNavigation`, etc.)
```

Comment:
Just wanted to confirm before merging this, should the changeset mention that clicks no longer trigger the tooltip, or that clicking a trigger once now dismisses the tooltip and cannot be toggled again until re-hovered? But not sure if it will still remain a patch then.

Reply:
You can phrase it like this:

```suggestion
Fixed an issue where `Tooltip` would stay open after clicking on the trigger element. The tooltip will now be correctly dismissed upon click.

Also affects other components that use tooltips internally: `IconButton`, `SideNavigation`, etc.
```